docs/tutorial.md

```diff
@@ -0,0 +1,721 @@
+the Blockstack iOS SDK through the CocoaPods dependency manager. You must use
+the `1.6.0.beta.1` version of CocoaPods or newer to avoid an incapability
+between Cocoapods and XCode.
+
+Before starting the tutorial, confirm you have installed CocoaPods.
+
+```bash
+$ pod --version
+1.6.0.beta.1
+```
+
+If you don't have the CocoaPods beta version, install it:
+
+```bash
+sudo gem install cocoapods -v 1.6.0.beta.1
+```
+
+### Use npm to install Yeoman and the Blockstack App Generator
+
+You use `npm` to install Yeoman. Yeoman is a generic scaffolding system that
+helps users rapidly start new projects and streamline the maintenance of
+existing projects.
+
+
+1. Install Yeoman.
+
+    ```bash
+    npm install -g yo
+    ```
+2. Install the Blockstack application generator.
+
+    ```bash
+    npm install -g generator-blockstack
+    ```
+## Build the Blockstack hello-world
+
+In this section, you build a Blockstack `hello-world` application. Then, you
+modify the `hello-world` to interact with the iOS app via a redirect.
+
+### Generate and launch your hello-blockstack application
+
+In this section, you build an initial React.js application called
+`hello-blockstack`.
+
+1. Create a `hello-blockstack` directory.
+
+    ```bash
+    mkdir hello-blockstack
+    ```
+
+2. Change into your new directory.
+
+    ```bash
+    cd hello-blockstack
+    ```
+
+3. Use Yeoman and the Blockstack application generator to create your initial `hello-blockstack` application.
+
+    ```bash
+    yo blockstack:react
+    ```
+
+    You should see several interactive prompts.
+
+    ```bash
+    $ yo blockstack:react
+    ==========================================================================
+    We are constantly looking for ways to make yo better!
+    May we anonymously report usage statistics to improve the tool over time?
+    More info: https://github.com/yeoman/insight & http://yeoman.io
+    ========================================================================== No
+
+         _-----_     ╭──────────────────────────╮
+        |       |    │      Welcome to the      │
+        |--(o)--|    │      Blockstack app      │
+       `---------´   │        generator!        │
+        ( _´U`_ )    ╰──────────────────────────╯
+        /___A___\   /
+         |  ~  |
+       __'.___.'__
+     ´   `  |° ´ Y `
+
+    ? Are you ready to build a Blockstack app in React? (Y/n)
+    ```
+
+4. Respond to the prompts to populate the initial app.
+
+    After the process completes successfully, you see a prompt similar to the following:
+
+    ```bash
+    [fsevents] Success:
+    "/Users/theuser/repos/hello-blockstack/node_modules/fsevents/lib/binding/Release/node-v59-darwin-x64/fse.node"
+    is installed via remote npm notice created a lockfile as package-lock.json.
+    You should commit this file. added 1060 packages in 26.901s
+    ```
+
+5. Run the initial application.
+
+    ```bash
+    npm start
```

Comment:
This did not open the browser and show the app automatically for me. Had to navigate to "localhost:8080/" on web browser.